roles/sosreport/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: sosreport - install required packages
+  become: true
+  ansible.builtin.package:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ required_packages_sosreport }}"
+
+- name: sosreport - remove temporary directory
+  become: true
+  ansible.builtin.file:
+    path: "{{ sosreport_tmpdir }}"
+    state: absent
+
+- name: sosreport - create required directories
+  become: true
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ operator_user }}"
+    group: "{{ operator_group }}"
+    mode: 0755
+  loop:
+    - "{{ sosreport_tmpdir }}"
+
+- name: sosreport - create local required directories
+  ansible.builtin.file:
+    path: "{{ sosreport_archive_directory }}"
+    state: directory
+    mode: 0755
+    recurse: true
+  delegate_to: localhost
+
+- name: sosreport - generate report
+  become: true
+  ansible.builtin.command: "sosreport --batch --tmp-dir {{ sosreport_tmpdir }} -z gzip --name {{ inventory_hostname_short }} -o {{ sosreport_plugins|join(',') }}"
+  changed_when: true
+
+- name: sosreport - rename md5 file
+  ansible.builtin.shell: "mv -f {{ sosreport_tmpdir }}/*.md5 {{ sosreport_tmpdir }}/{{ sosreport_archive_filename }}.md5"
+  changed_when: true
```

Comment:
Is it possible to do a move with an Ansible task?

Reply:
Found this: https://stackoverflow.com/questions/24162996/how-to-move-rename-a-file-using-an-ansible-task-on-a-remote-system
Up until now there isn´t another way.
But i think i will use the command module.